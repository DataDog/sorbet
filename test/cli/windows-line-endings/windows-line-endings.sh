#!/bin/bash

file $(realpath test/cli/windows-line-endings/windows-line-endings.rb)

main/sorbet --silence-dev-message test/cli/windows-line-endings/windows-line-endings.rb 2>&1
