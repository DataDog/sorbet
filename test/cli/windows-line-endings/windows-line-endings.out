/pay/src/sorbet/test/cli/windows-line-endings/windows-line-endings.rb: ASCII text, with CRLF line terminators
No errors! Great job.
